crates/aranya-daemon-api/src/service.rs

```diff
@@ -132,6 +132,22 @@ impl From<Id> for AfcId {
 // serialized command which must be passed over AFC.
 pub type AfcCtrl = Vec<Box<[u8]>>;
 
+/// Configuration values for syncing with a peer
+#[derive(Serialize, Deserialize, Clone, Copy, Debug)]
+pub struct SyncPeerConfig {
```

Comment:
A couple of things:

1. `aranya-client` shouldn't expose the `aranya-daemon` RPC type. It should have its own config and convert into an internal RPC type.
2. Given (1), let's make `aranya-client`'s config type use the builder pattern: We're going to need to support a number of different arguments in the future and the builder pattern lets us make API changes _much_ more easily.

```rust 
pub struct PeerSyncConfig {}
impl PeerSyncConfig {
    pub fn builder() -> PeerSyncConfigBuilder { }
}
pub struct PeerSyncConfigBuilder { }
impl PeerSyncConfigBuilder {
    pub fn build() -> PeerSyncConfig { }
}
```